core/src/com/unciv/logic/civilization/PolicyManager.kt

```diff
@@ -129,11 +129,18 @@ class PolicyManager {
 
     fun isAdopted(policyName: String): Boolean = adoptedPolicies.contains(policyName)
 
-    fun isAdoptable(policy: Policy): Boolean {
+    /**
+     * Test whether a policy is adoptable according to the RuleSet (ignoring cost).
+     * Note: branch completion policies are automatic and therefore not adoptable in this test.
+     * @param policy The Policy to check
+     * @param testEra Include era test (with false the function returns whether the policy is adoptable now or in the future)
+     * @return `true` if the policy can be adopted, `false` if some rule prevents it (including when it's already adopted) 
+     */
+    fun isAdoptable(policy: Policy, testEra: Boolean = true): Boolean {
```

Comment:
'test' has a very specific meaning - it means out-of-game tests of the code. Such a variable would indicate that the era is 
a 'fake' era or something similar. This should be renamed to 'checkEra'.